src/cmdstan/command.hpp

```diff
@@ -242,11 +242,14 @@ namespace stan {
 
       Eigen::VectorXd cont_params = Eigen::VectorXd::Zero(model.num_params_r());
 
+      parser.print(info);
+      info();
+      
       if (output_stream) {
         io::write_stan(output_stream, "#");
         io::write_model(output_stream, model.model_name(), "#");
-        parser.print(info);
-        info();
+        interface_callbacks::writer::stream_writer sample_writer(*output_stream, "#");
+        parser.print(sample_writer);
```

Comment:
I take it this is intentional --- only write to `sample_writer` if `output_stream` isn't `NULL`?